content/rest-api/update.md

```diff
@@ -313,15 +313,15 @@ You want to remove the product `boots-4846` from the category `boots`. Here is h
 
 ## Patch product values
 
-The PATCH behavior described above is quite intuitive. However, applying a PATCH containing product values on a product is a bit different.
+The PATCH behavior described above is quite intuitive. However, applying a PATCH containing [product values](/documentation/resources.html#product-values) on a product is a bit different.
 
 ::: info
 In the examples below only products values are represented, but usually products also include other information as specified in the standard format.
 :::
 ::: panel-link Don't know what a product value is? [Check the product value section](/documentation/resources.html#product-values)
 :::
 
-### Add a product value
+### Add a product value or a reference entity record value
 You want to add the `description` of the product `boots-4846` for the `en_US` locale.
```

Comment:
so, adding a value is a usecase ;) (not the scope of this PR, but definitively something to discuss, I forgot that we documented it this way)

Reply:
Removed